src/styles/style.css

```diff
@@ -0,0 +1,122 @@
+@import "./star.css";
+
+:root {
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
+
+  /* button bg and :hover border+text color */
+  --button-bg-color: #00acdc;
+
+  --button-text-color: #fff;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
```

Comment:
you should not have any paddings in this task. it is why you don't pass tests. just play with margins and flex.

Reply:
The test has been passed.
(There are some more commits after the PR itself, so I have resolved it already).

The test failed due to absence of `display:block` in the `img` tag.

Reply:
The test has been passed.
(There are some more commits after the PR itself, so I have resolved it already).

The test failed due to absence of display:block in the img tag.

I just forgot to deploy the new version :(